### PR TITLE
Document current callback retry logic.

### PR DESCRIPTION
**Pull request by jviso**

### CODE REVIEW

**Comment by jviso** [comment]
@briancbarrow, @SandraRodgers, @shirgoldbird -- could I get your eyes on this? Thanks!

File: developers/src/pages/api-reference/transcription.mdx [hunk 1]
```diff
@@ -2516,4 +2518,4 @@ establishConnection()
 </CodeGroup>
 
 </div>
-</ColumnsForCode>
\ No newline at end of file
+</ColumnsForCode>
```
Comment (jviso) [inline developers/src/pages/api-reference/transcription.mdx:2521]: Not sure what this is about. git being weird, I think?

File: developers/src/pages/api-reference/transcription.mdx [hunk 2]
```diff
@@ -698,7 +698,9 @@ Deepgram does not store transcriptions. Make sure to save output or [return tran
 			Notes:
 			<ul class="info">
 				<li>You may embed basic authentication credentials in the callback URL.</li>
-				<li>Only ports 80, 443, 8080, and 8443 can be used for callbacks</li>
+				<li>Only ports 80, 443, 8080, and 8443 can be used for callbacks.</li>
+				<li>If the HTTP status code of the response to the callback POST request is not 200-299,
```
Comment (shirgoldbird) [inline developers/src/pages/api-reference/transcription.mdx:702]: What about:

> If the HTTP status code of the response to the callback POST request is unsuccessful (not 200-299)

Makes the reader have to think less :)

File: developers/src/pages/documentation/features/callback/index.mdx [hunk 3]
```diff
@@ -88,6 +88,7 @@ Depending on the type of submitted audio, the response will vary.
 ### Pre-recorded Audio
 
 When Deepgram has finished analyzing the audio, it will send a `POST` request to the provided callback URL with an appropriate HTTP status code.
+If the HTTP status code of the response to the callback POST request is not 200-299, Deepgram will retry the callback up to 10 times with a 30 second delay between attempts.
```
Comment (shirgoldbird) [inline developers/src/pages/documentation/features/callback/index.mdx:91]: Same suggestion here

**Review by shirgoldbird — APPROVED** [review]
LGTM, suggested one small change

**Comment by jviso** [comment]
Thanks, @shirgoldbird! I made that change and added the same line in one other place I overlooked originally. Do you think the line does indeed belong in the callback "guide" as well? IDK the intended differences between the feature-specific docs and the guides.

**Comment by shirgoldbird** [comment]
@jviso that guide cross-links to the features page in the first sentence so it seems likely devs will read both. I vote for merging this for the sake of expedience/ease-in-PR-submission process 🙏🏻

**Pull request merged**